maintenance/wikia/cronjobs/resetWeeklyUserContributionsCount.php

```diff
@@ -1,19 +1,18 @@
 <?php
-
-use \Wikia\Logger\WikiaLogger;
-
 /**
  * On Special:Community we have weekly users rank with most contributions.
- * We need to reset it after every Sunday
+ * We need to reset it after every Sunday.
+ *
+ * @group cronjobs
+ * @see reset-weekly-user-contributions.yaml
  */
 
-require_once( __DIR__ . '/../../Maintenance.php' );
+require_once __DIR__ . '/../../Maintenance.php';
 
-/**
- * Class ResetWeeklyUserContributionsCount
- */
 class ResetWeeklyUserContributionsCount extends Maintenance {
 
+	use Wikia\Logger\Loggable;
```

Comment:
This will fail with `PHP Fatal error:  Trait 'Wikia\Logger\Loggable' not found` because according to @macbre (I couldn't find it in the code) the classes are autoloaded just before `execute()` is called. See https://github.com/Wikia/app/pull/15797/commits/fffa65184ca0368d7030f8de6bbe9cc747928f32.